docs/CLI.md

```diff
@@ -118,7 +118,7 @@ import TOCInline from '@theme/TOCInline';
 
 When you run `jest` with an argument, that argument is treated as a regular expression to match against files in your project. It is possible to run test suites by providing a pattern. Only the files that the pattern matches will be picked up and executed. Depending on your terminal, you may need to quote this argument: `jest "my.*(complex)?pattern"`. On Windows, you will need to use `/` as a path separator or escape `\` as `\\`.
 
-### `--bail`
+### `--bail=n`
```

Comment:
Perhaps it would be more precise to mark the argument as optional? See `--coverage` below.

```suggestion
### `--bail[=<n>]`
```

Reply:
That's correct, thanks ! I've just checked other options and indeed the doc used to add brackets for optional values (like `--coverage[=<boolean>]`).